epytope/Data/pssms/syfpeithi/mat/DRB1_1101_9.py

```diff
@@ -1 +1 @@
\ No newline at end of file
+DRB1_1101_9 = {0: {'A': 0, 'C': 0, 'E': 0, 'D': 0, 'G': 0, 'F': 0, 'I': 6, 'H': 0, 'K': 0, 'M': 6, 'L': 6, 'N': 0, 'Q': 0, 'P': 0, 'S': 0, 'R': 0, 'T': 0, 'W': 10, 'V': 6, 'Y': 10, 'X': 0}, 1: {'A': 0, 'C': 0, 'E': 0, 'D': 0, 'G': 0, 'F': 10, 'I': 0, 'H': 0, 'K': 0, 'M': 0, 'L': 0, 'N': 0, 'Q': 0, 'P': 0, 'S': 0, 'R': 0, 'T': 2, 'W': 0, 'V': 0, 'Y': 0, 'X': 0}, 2: {'A': 0, 'C': 0, 'E': 0, 'D': 0, 'G': 0, 'F': 1, 'I': 0, 'H': 0, 'K': 0, 'M': 0, 'L': 0, 'N': 0, 'Q': 0, 'P': 0, 'S': 0, 'R': 0, 'T': 0, 'W': 0, 'V': 1, 'Y': 0, 'X': 0}, 3: {'A': 0, 'C': 0, 'E': 0, 'D': 0, 'G': 0, 'F': 0, 'I': 6, 'H': 0, 'K': 0, 'M': 6, 'L': 6, 'N': 0, 'Q': 0, 'P': 0, 'S': 0, 'R': 0, 'T': 0, 'W': 0, 'V': 6, 'Y': 0, 'X': 0}, 4: {'A': 0, 'C': 0, 'E': 0, 'D': 0, 'G': 0, 'F': 0, 'I': 0, 'H': 0, 'K': 0, 'M': 0, 'L': 0, 'N': 0, 'Q': 0, 'P': 0, 'S': 0, 'R': 0, 'T': 0, 'W': 0, 'V': 2, 'Y': 0, 'X': 0}, 5: {'A': 0, 'C': 0, 'E': 0, 'D': 0, 'G': 0, 'F': 0, 'I': 0, 'H': 8, 'K': 8, 'M': 0, 'L': 0, 'N': 0, 'Q': 0, 'P': 0, 'S': 0, 'R': 8, 'T': 0, 'W': 0, 'V': 0, 'Y': 0, 'X': 0}, 6: {'A': 0, 'C': 0, 'E': 0, 'D': 0, 'G': 0, 'F': 0, 'I': 0, 'H': 0, 'K': 0, 'M': 0, 'L': 0, 'N': 0, 'Q': 0, 'P': 0, 'S': 0, 'R': 1, 'T': 0, 'W': 0, 'V': 0, 'Y': 0, 'X': 0}, 7: {'A': 0, 'C': 0, 'E': 0, 'D': 0, 'G': 0, 'F': 0, 'I': 1, 'H': 0, 'K': 1, 'M': 0, 'L': 1, 'N': 0, 'Q': 0, 'P': 0, 'S': 0, 'R': 2, 'T': 0, 'W': 0, 'V': 1, 'Y': 0, 'X': 0}, 8: {'A': 6, 'C': 0, 'E': 0, 'D': 0, 'G': 6, 'F': 0, 'I': 0, 'H': 0, 'K': 0, 'M': 0, 'L': 0, 'N': 0, 'Q': 0, 'P': 6, 'S': 6, 'R': 0, 'T': 0, 'W': 0, 'V': 0, 'Y': 0, 'X': 0}}
```

Comment:
Not sure what has changed here.

Reply:
This is confusing. Nothing changed here. I double checked both versions in a text editor with the find function..